src/components/Garden/Home.js

```diff
@@ -121,7 +121,7 @@ const Home = React.memo(function Home() {
                   margin: ${(below('medium') ? 0 : 3) * GU}px;
                 `}
               >
-                {!compactMode && (
+                {layoutName !== 'small' && (
```

Comment:
I wonder how we should display this information on mobile. Maybe we can use the [accordion component ](https://ui.aragon.org/accordion/) 🤔

Reply:
https://user-images.githubusercontent.com/9082013/131398930-2711389a-f7c1-4498-88a2-221ac51b87a6.mov

Reply:
I like this. Let's address in another PR?